.travis.yml

```diff
@@ -18,7 +18,6 @@ env:
     - TARGETS="-C libbeat testsuite"
     - TARGETS="-C topbeat testsuite"
     - TARGETS="-C filebeat testsuite"
-    - TARGETS="-C winlogbeat testsuite"
```

Comment:
The system tests are all (now) skipped unless the OS is windows so I want it removed to use less Travis resources.